ChatApp/Presentation/View/ViewController/MessageTableViewCell/MessageTableViewCell.swift

```diff
@@ -0,0 +1,257 @@
+//
+//  MessageTableViewCell.swift
+//  ChatApp
+//
+//  Created by Nika Gogichashvili on 03.05.23.
+//
+
+import UIKit
+
+class MessageTableViewCell: UITableViewCell {
+    
+    // MARK: Variable
+    static let reuseIdentifier = "MessageTableViewCell"
+    var leadingConstraint = NSLayoutConstraint()
+    var trailingConstraint = NSLayoutConstraint()
+    
+    // MARK: Property
+    private lazy var mainBubbleView: UIView = {
+        let bubble = UIView()
+        bubble.layer.cornerRadius = Constants.MainBubleViewLeft.cornerRadius
+        bubble.layer.masksToBounds = true
+        bubble.backgroundColor = .gray
+        return bubble
+    }()
+    
+    private lazy var messageLabel: UILabel = {
+        let label = UILabel()
+        label.font = Constants.MessageLabelLeft.textFontSize
+        label.textColor = .white
+        return label
+    }()
+    
+    private lazy var dateLabel: UILabel = {
+        let label = UILabel()
+        label.font = Constants.DateLabelLeft.textFontSize
+        label.textColor = .gray
+        return label
+    }()
+    
+    private lazy var mediumBubbleView: UIView = {
+        let bubble = UIView()
+        bubble.layer.cornerRadius = Constants.MediumBubleViewLeft.radius
+        bubble.layer.masksToBounds = true
+        bubble.backgroundColor = .gray
+        return bubble
+    }()
+    
+    private lazy var smallBubbleView: UIView = {
+        let bubble = UIView()
+        bubble.layer.cornerRadius = Constants.SmallBubleViewLeft.radius
+        bubble.layer.masksToBounds = true
+        bubble.backgroundColor = . gray
+        return bubble
+    }()
+    
+    
+    // MARK: init
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        self.backgroundColor = .clear
+        selectionStyle = .none
+        
+        setUp()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: Configure
+    func configure(with message: String, indexpath: IndexPath) {
+        messageLabel.text = message
+       
+        if indexpath.row == 0 {
+            setUpLeftConstraint()
+        } else if indexpath.row == 1 {
+            setUpRightConsstraint()
+        } else if indexpath.row == 2 {
+            setUpLeftConstraint()
+        }
```

Comment:
You can combine `indexpath.row == 0` and `indexpath.row == 2` with help of OR operator, cause they have the same result.